README.md

```diff
@@ -1,7 +1,21 @@
 ![SPHERE.IO icon](https://admin.sphere.io/assets/images/sphere_logo_rgb_long.png)
 
-# Product import
+# Sphere Product Import
 
 [![NPM](https://nodei.co/npm/sphere-product-import.png?downloads=true)](https://www.npmjs.org/package/sphere-product-import)
 
 [![Build Status](https://secure.travis-ci.org/sphereio/sphere-product-import.png?branch=master)](http://travis-ci.org/sphereio/sphere-product-import) [![NPM version](https://badge.fury.io/js/sphere-product-import.png)](http://badge.fury.io/js/sphere-product-import)
+
+This is a library with modules for importing and managing product data in sphere. 
```

Comment:
I already changed it this morning https://github.com/sphereio/sphere-product-import/blob/master/README.md